resource/chatadapter.go

```diff
@@ -27,7 +27,7 @@ func sendToChat(
 	instanceVars := env.Get("BUILD_PIPELINE_INSTANCE_VARS")
 	buildURL := concourseBuildURL(atc, team, pipeline, job, buildN, instanceVars)
 
-	webhook, ok := source["gchat_webhook"].(string)
```

Comment:
leftovers

Reply:
Is this the only change needed? `gchat_webhook` is declared in source config. Why does it still work if you use `gchatWebhookKey`?

Reply:
Because `gchatWebhookKey` has the value `"gchat_webhook"` :-)

Reply:
> Because `gchatWebhookKey` has the value `"gchat_webhook"` :-)

Ok. Now its clear. Thank
https://github.com/Pix4D/cogito/blob/fa8446007ec2314c4768182ca0f2b40945438d6a/resource/resource.go#L27